sdk/program/src/program_stubs.rs

```diff
@@ -30,6 +30,10 @@ pub trait SyscallStubs: Sync + Send {
     fn sol_log_compute_units(&self) {
         sol_log("SyscallStubs: sol_log_compute_units() not available");
     }
+    fn sol_remaining_compute_units(&self) -> u64 {
+        sol_log("SyscallStubs: sol_remaining_compute_units() defaulting to u64::MAX");
+        u64::MAX
```

Comment:
total nit: 0 instead of `u64::MAX` feels like a better stub value here, as that'd cause any programs under test that are looping on the syscall to break instead of infinite looping.

Reply:
Changed